packages/server/graphql/public/types/Company.ts

```diff
@@ -20,6 +21,8 @@ const getSuggestedTierOrganizations = async (
   const organizations = await dataLoader.get('organizationsByActiveDomain').load(domain)
   const orgIds = organizations.map(({id}) => id)
   const viewerId = getUserId(authToken)
+  const viewer = await getUserById(viewerId)
```

Comment:
+1 how can i test this? to fetch `{viewer { company { id } }` we already have to get the viewer, so I'm not sure why it's needed? if it is needed, could we use the dataloader to reduce a fetch?